docs/en/deployment/how_to_use_on_kubernetes.md

```diff
@@ -106,7 +106,7 @@ juicefs-sc   csi.juicefs.com   Retain          Immediate           false
 
 #### Install with kubectl
 
-Since Kubernetes will discard some old APIs during the version change process, you need to select the applicable deployment file according to the version of Kubernetes you are using:
+Since Kubernetes deprecates some old APIs during the version bumping process, you need to select the applicable deployment file fit for the version of Kubernetes you are using:
```

Comment:
```suggestion
Since Kubernetes deprecates some old APIs during the version bumping process, you need to select the applicable deployment file that works for the version of Kubernetes you are using:
```